go/lesson03/exercise/client/hello.go

```diff
@@ -47,6 +48,7 @@ func formatString(ctx context.Context, helloTo string) string {
 
 	resp, err := xhttp.Do(req)
 	if err != nil {
+		ext.Error.Set(span, true)
```

Comment:
change this to ext.LogError ?

Reply:
This will be removed since we leave it as exercise. Users should follow readme to update the code.